tasks/scheduler.go

```diff
@@ -49,6 +74,28 @@ func NewScheduler(workers int, deliverTxFunc func(ctx sdk.Context, req types.Req
 	}
 }
 
+func (s *scheduler) invalidateTask(task *deliverTxTask) {
+	for _, mv := range s.multiVersionStores {
+		mv.InvalidateWriteset(task.Index, task.Incarnation)
+	}
+}
+
+func (s *scheduler) findConflicts(task *deliverTxTask) []int {
+	var conflicts []int
+	uniq := make(map[int]struct{})
+	for _, mv := range s.multiVersionStores {
+		mvConflicts := mv.ValidateTransactionState(task.Index)
+		for _, c := range mvConflicts {
```

Comment:
note: conflicts are already deduped + sorted, so this is unnecessary

Reply:
it's necessary to dedupe/sort across storekeys